MantidQt/SliceViewer/inc/MantidQtSliceViewer/PeakPrimitives.h

```diff
@@ -0,0 +1,94 @@
+#ifndef MANTID_SLICEVIEWER_PEAK_PRIMITIVES_H
+#define MANTID_SLICEVIEWER_PEAK_PRIMITIVES_H
+
+#include "MantidKernel/V3D.h"
+
+namespace MantidQt
+{
+namespace SliceViewer
+{
+
+struct DLLExport PeakPrimitives {
```

Comment:
I just spotted this when I was having a quick look out of curiousity.
I think we are using the generic DLLExport in 90% of places or more, where we should be using the library specific dll export macros (EXPORT_OPT_MANTIDQT_SLICEVIEWER) in this case.

This might need a bit of maintenance work or might get fixed together with the code reorg.